.gitignore

```diff
@@ -357,4 +357,4 @@ MigrationBackup/
 
 # Community Toolkit Labs generated files
 Toolkit.Labs.All.sln
-Labs.SampleRefs.props
+common/MultiTarget/Generated/**
```

Comment:
We'll be changing this again soon after we move to a submodule.

When we move to a submodule, we won't need to use `.gitignore` or `git update-index --assume-unchanged` on anything in the common folder, since submodules have a separate working tree.